astropy/cosmology/core.py

```diff
@@ -1825,7 +1825,7 @@ def efunc(self, z):
         return np.sqrt(zp1 ** 3 * (Or * zp1 + Om0) + Ode0)
 
     def inv_efunc(self, z):
-        """Function used to calculate :math:`\frac{1}{H_z}`.
+        """Function used to calculate :math:`\\frac{1}{H_z}`.
```

Comment:
This one is broken.  But maybe instead the string should be converted to a raw string.  Or better still, `LambdaCDM` could use [`InheritDocstrings`](http://docs.astropy.org/en/latest/api/astropy.utils.misc.InheritDocstrings.html?highlight=inheritdocstrings)